cm-mlops/script/app-mlperf-inference-reference/customize.py

```diff
@@ -60,7 +60,7 @@ def preprocess(i):
     if 'CM_MLPERF_LOADGEN_MAX_BATCHSIZE' in env:
         env['CM_MLPERF_LOADGEN_EXTRA_OPTIONS'] += " --max-batchsize " + env['CM_MLPERF_LOADGEN_MAX_BATCHSIZE']
 
-    if 'CM_MLPERF_LOADGEN_QUERY_COUNT' in env:
+    if 'CM_MLPERF_LOADGEN_QUERY_COUNT' in env and '3d-unet' not in env['CM_MODEL']:
```

Comment:
I have a feeling that we should move '3d-unet' into a list and add it to meta because I guess some other models/tasks may not have ---count applicable to them ...

Reply:
Thank you @gfursin for the review. I'm now using a tmp env variable for excluding query_counts when not applicable.